modules/hitlnext/src/views/full/app/Reducer.ts

```diff
@@ -78,6 +79,14 @@ const Reducer = (state: IState, action: ActionType): IState => {
       return produce(state, draft => {
         draft.error = action.payload
       })
+    case 'removeHandoff':
+      return produce(state, draft => {
+        const handoffsCopy = { ...state.handoffs }
```

Comment:
that doesn't really create a deep copy. can't you simply `delete draft.handoffs[action.payload]` would that work ?

Reply:
I does, thanks for the suggestion!